run-container.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Requires
+# kube config file mounted to /usr/tnf/kubeconfig/config
+# TNF config files mounted into /usr/tnf/config
+# A directory to output claim into mounted at /usr/tnf/claim
+
+
+usage() {
+	echo "$0 -k KUBECONFIG -t TNFCONFIG -o OUTPUT_LOC SUITE [... SUITE]"
+	echo "Configure and run the containerised TNF test offering."
+	echo "  e.g."
+	echo "    $0 -k ~/.kube/config -t ~/tnf/config -o ~/tnf/output diagnostic generic"
+	echo "  will run the diagnostic and generic tests, and output the result into"
+	echo "  ~/tnf/output"
+	echo ""
+	echo "Allowed tests are listed in the README."
+	echo "Note: Tests muse be specified after all other arguments"
+
+}
+
+usage_error() {
+	usage
+	exit 1
+}
+
+CONTAINER_TNF_DIR=/usr/tnf
+
+if (($# < 7)); then
+	usage_error
+fi;
+
+# Parge args beginning with -
+while [[ $1 == -* ]]; do
+	echo $1 $2
+    case "$1" in
+      -h|--help|-\?) usage; exit 0;;
+      -k) if (($# > 1)); then
```

Comment:
As a comment but not a requirement, we could make `-k` optional.  My idea is that if `-k` is specified, then it takes precedence.  If `-k` is not specified, `$KUBECONFIG` is fed in from the underlying host.  If `-k` is not specified, and `$KUBECONFIG` is not defined, then it is an error case.  This may be a story we could create and add to the backlog for later if time permits.

Reply:
Absolutely. I considered the same but decided it was an unnecessary step at this point given time spent on the story. I'll document it in a new story.